debian/changelog

```diff
@@ -1,3 +1,9 @@
+tiqit (1.0.6-1) trusty; urgency=low
```

Comment:
Version number also needs updating in setup.py & scripts/tiqit/__init__.py

Reply:
Fixed